src/utils/runtime/src/lib.rs

```diff
@@ -138,7 +140,7 @@ pub fn init_risingwave_logger(settings: LoggerSettings) {
         #[cfg(debug_assertions)]
         let filter = filter.with_default(Level::DEBUG);
 
-        layers.push(fmt_layer.with_filter(filter).boxed());
+        layers.push(fmt_layer.with_filter(to_env_filter(filter)).boxed());
```

Comment:
Does this work?

```rust
let filter = filter.or(EnvFilter::from_default_env())
```

Reply:
`and` works, thanks.

Reply:
But it's not the same as `to_env_filter`.

- `to_env_filter` is overwriting using RUST_LOG.
- `filter.and ` cannot lower the predefined filter level in `init_risingwave_logger`.

Reply:
I guess we can always make the default configuration stricter, and allow relaxing with env var? When `RUST_LOG` is not set, the `EnvFilter` fall back to `ERROR`. So `env_filter.or(targets_filter)` should work. 🤔